nrf_wifi/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -88,6 +70,217 @@
+	/*! Systematic adjustment value corresponding to Channel 165*/
+	signed char syst_off_hb_high_chan;
+} __NRF_WIFI_PKD;
+
+/*! Max TX power value for which both EVM and SEM pass */
+struct nrf_wifi_tx_pwr_ceil
+{
+	/*! Max output power for 11b for channel 7  */
+	signed char max_dsss_pwr;
+	/*! Max output power for MCS7 for channel 7  */
+	signed char max_lb_mcs7_pwr;
+	/*! Max output power for MCS0 for channel 7  */
+	signed char max_lb_mcs0_pwr;
+	/*! Max output power for MCS7 for channel 36  */
+	signed char max_hb_low_chan_mcs7_pwr;
+	/*! Max output power for MCS7 for channel 100  */
+	signed char max_hb_mid_chan_mcs7_pwr;
+	/*! Max output power for MCS7 for channel 165  */
+	signed char max_hb_high_chan_mcs7_pwr;
+	/*! Max output power for MCS0 for channel 36  */
+	signed char max_hb_low_chan_mcs0_pwr;
+	/*! Max output power for MCS0 for channel 100  */
+	signed char max_hb_mid_chan_mcs0_pwr;
+	/*! Max output power for MCS0 for channel 165  */
+	signed char max_hb_high_chan_mcs0_pwr;
+} __NRF_WIFI_PKD;
+
+/*! RX gain adjustment offsets */
+struct nrf_wifi_rx_gain_offset
+{
+	/*! Channel 7 */
+	signed char rx_gain_lb_chan;
+	/*! Channel 36 */
+	signed char rx_gain_hb_low_chan;
+	/*! Channel 100 */
+	signed char rx_gain_hb_mid_chan;
+	/*! Channel 165 */
+	signed char rx_gain_hb_high_chan;
+} __NRF_WIFI_PKD;
+
+/*! Voltage and Temperature dependent backoffs */
+struct nrf_wifi_temp_volt_depend_params
+{
+	/*! maximum chip temperature in Centigrade*/
+	signed char max_chip_temp;
+
+	/*! minimum chip temperature in Centigrade*/
+	signed char min_chip_temp;
+
+	/*! TX power backoff at high temperature in 2.4GHz */
+	signed char lb_max_pwr_bkf_hi_temp;
+
+	/*! TX power backoff at low temperature in 2.4GHz */
+	signed char lb_max_pwr_bkf_low_temp;
+
+	/*! TX power backoff at high temperature in 5GHz */
+	signed char hb_max_pwr_bkf_hi_temp;
+
+	/*! TX power backoff at low temperature in 5GHz */
+	signed char hb_max_pwr_bkf_low_temp;
+
+	/*! Voltage back off value in  LowBand when VBAT< VBAT_VERYLOW */
+	signed char lb_vbt_lt_vlow;
+
+	/*! Voltage back off value in  HighBand when VBAT< VBAT_VERYLOW */
+	signed char hb_vbt_lt_vlow;
+
+	/*! Voltage back off value in  LowBand when VBAT< VBAT_LOW */
+	signed char lb_vbt_lt_low;
+
+	/*! Voltage back off value in  HighBand when VBAT< VBAT_LOW */
+	signed char hb_vbt_lt_low;
+
+	/*! Reserved bytes */
+	signed char reserved[4];
+
+} __NRF_WIFI_PKD;
+
+struct nrf_wifi_phy_rf_params
+{
+	unsigned char reserved[6];
+	struct nrf_wifi_xo_freq_offset xo_offset;
+	struct nrf_wifi_pd_adst_val pd_adjust_val;
+	struct nrf_wifi_tx_pwr_systm_offset syst_tx_pwr_offset;
+	struct nrf_wifi_tx_pwr_ceil max_pwr_ceil;
+	struct nrf_wifi_rx_gain_offset rx_gain_offset;
+	struct nrf_wifi_temp_volt_depend_params temp_volt_backoff;
+} __NRF_WIFI_PKD;
+
+enum RF_PARAMS_OFFSETS
+{
+	NRF_WIFI_XO_FREQ_BYTE_OFFSET = (6),
+	NRF_WIFI_PD_ADST_VAL_BYTE_OFFSET = (NRF_WIFI_XO_FREQ_BYTE_OFFSET					+ sizeof(struct nrf_wifi_xo_freq_offset  ) ),
```

Comment:
is this working for you? Enumerations must either be int or long, if you want to use an offset, you need to use macros to have sizeof.

Reply:
Yes it is working. I tested it.

Reply:
This piece of code is not used so I did not face any issue while testing. I will delete this code.